libs/common-const/src/nativeAndWrappedTokens.ts

```diff
@@ -1,62 +1,84 @@
-import { SupportedChainId, SupportedChainId as ChainId } from '@cowprotocol/cow-sdk'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { TokenWithLogo } from './types'
-import { WETH9 } from '@uniswap/sdk-core'
 import { cowprotocolTokenLogoUrl } from './cowprotocolTokenLogoUrl'
 
-// See https://github.com/cowprotocol/contracts/commit/821b5a8da213297b0f7f1d8b17c893c5627020af#diff-12bbbe13cd5cf42d639e34a39d8795021ba40d3ee1e1a8282df652eb161a11d6R13
-export const NATIVE_CURRENCY_BUY_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
+export const DEFAULT_NATIVE_CURRENCY_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
 
-const weth9Mainnet = WETH9[SupportedChainId.MAINNET]
-export const WETH_MAINNET = new TokenWithLogo(
-  cowprotocolTokenLogoUrl(weth9Mainnet.address.toLowerCase(), SupportedChainId.MAINNET),
-  weth9Mainnet.chainId,
-  weth9Mainnet.address,
-  weth9Mainnet.decimals,
-  weth9Mainnet.symbol,
-  weth9Mainnet.name
-)
-// xDAI tokens
-export const WXDAI = new TokenWithLogo(
-  undefined,
-  ChainId.GNOSIS_CHAIN,
-  '0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d',
-  18,
-  'WXDAI',
-  'Wrapped XDAI'
-)
-export const WETH_GOERLI = new TokenWithLogo(
-  WETH_MAINNET.logoURI,
-  ChainId.GOERLI,
-  '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
-  18,
-  'WETH',
-  'Wrapped Görli Ether'
-)
-export const WETH_SEPOLIA = new TokenWithLogo(
-  WETH_MAINNET.logoURI,
-  ChainId.SEPOLIA,
-  '0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14',
-  18,
-  'WETH',
-  'Wrapped Ether'
-)
-export const WRAPPED_NATIVE_CURRENCY: Record<SupportedChainId, TokenWithLogo> = {
-  [SupportedChainId.MAINNET]: WETH_MAINNET,
-  [SupportedChainId.GNOSIS_CHAIN]: WXDAI,
-  [SupportedChainId.GOERLI]: WETH_GOERLI,
-  [SupportedChainId.SEPOLIA]: WETH_SEPOLIA,
+const DEFAULT_NATIVE_DECIMALS = 18
+const ETH_MAINNET_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
```

Comment:
This is not ETH!
And actually, I think it's important we remain aware it's WETH9 and not [WETH10](https://ethereum-magicians.org/t/rfc-improving-weth9-moving-to-a-better-wrapped-ether-implementation/12487/1), even though there's no WETH10 adoption AFAICT.

```suggestion
const WETH9_MAINNET_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
```

Reply:
Nice catch, thanks! FIxed